content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss

```diff
@@ -36,6 +36,7 @@ $excerpt-line-height: 20;
     flex-direction: column;
     flex-grow: 1;
     padding: 16px;
+    overflow-wrap: anywhere;
```

Comment:
Helps to wrap long words in a story that can break alignment which will have high probability once we start supporting localization e.g. I'm a Looooooooooooooooooonngg word